amundsen_application/static/js/components/SearchPage/index.tsx

```diff
@@ -110,6 +119,13 @@ export class SearchPage extends React.Component<SearchPageProps, SearchPageState
     };
   };
 
+  getSanitizedUrlParams = (searchTerm: string, pageIndex: number, selectedTab: ResourceType) => {
+    const currentTab = this.getSelectedTabByResourceType(selectedTab);
```

Comment:
nit: It would appear this method could also have isolated the following shared logic as well
```
const params = qs.parse(this.props.location.search);
const { searchTerm, pageIndex, selectedTab } = params;
```